features/steps/steps.go

```diff
@@ -114,7 +114,7 @@ func (c *Component) theFollowingInstanceIsAvailable(id string, instance *godog.D
 }
 
 // theFollowingCantabularCategoriesAreAvailable generates a mocked response for Cantabular Server
-// GET /v9/codebook/{name} with the provided query
+// GET /v1/codebook/{name} with the provided query
```

Comment:
Is this `v1` correct? Shouldn't it be `v10`?

Reply:
Yes :) this was a typo which I have now fixed